iphone/cli/commands/_build.js

```diff
@@ -3562,32 +3562,55 @@ iOSBuilder.prototype.createXcodeProject = function createXcodeProject(next) {
 						extBuildSettings.CODE_SIGN_IDENTITY = buildSettings.CODE_SIGN_IDENTITY;
 					}
 
+					const setEntitlementsFile = entFile => {
+						let src = path.join(ext.basePath, entFile);
+						if (fs.existsSync(src)) {
+							extBuildSettings.CODE_SIGN_ENTITLEMENTS = '"' + path.join(ext.relPath, entFile) + '"';
+							targetInfo.entitlementsFile = path.join(this.buildDir, ext.relPath, entFile);
+						} else {
+							src = path.join(ext.basePath, targetName, entFile);
+							if (fs.existsSync(src)) {
+								extBuildSettings.CODE_SIGN_ENTITLEMENTS = '"' + path.join(ext.relPath, targetName, entFile) + '"';
+								targetInfo.entitlementsFile = path.join(this.buildDir, ext.relPath, targetName, entFile);
+							} else {
+								delete extBuildSettings.CODE_SIGN_ENTITLEMENTS;
```

Comment:
I think we should spit out a warning here that we couldn't resolve the configured entitlements file instead of just silently deleting the `CODE_SIGN_ENTITLEMENTS` setting.

Reply:
That's what I originally thought too, but I realized that not every extension target has entitlements. For example, File Provider UI targets do not have entitlements. So, having a warning leads to false positives and that would be confusing.

Reply:
True, but wouldn't `CODE_SIGN_ENTITLEMENTS` then be empty anyway and we could check for that?

Reply:
We do! https://github.com/appcelerator/titanium_mobile/pull/10813/files#diff-f430483233aa01af5b10df8390f9635dR3582

Reply:
Yes, so in that specific case it should be fine to print a warning, shouldn't it? If {{CODE_SIGN_ENTITLEMENTS}} is explicitly set, warn if we can't properly resolve it. If it's not set, e.g. for the File Provider UI extension, that code path is ignored anyway.

Reply:
I added a warning. Should be good to go now.